signimage/Makefile

```
@@ -0,0 +1,34 @@
+IMAGE_TAG_BASE ?= quay.io/chrisp262/kmod-signer
+IMAGE_TAG ?= $(shell  git log --format="%H" -n 1)
+# Image URL to use all building/pushing image targets
+IMG ?= $(IMAGE_TAG_BASE):$(IMAGE_TAG)
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+.PHONY: all
+all: signimage
+
+.PHONY: fmt
+fmt: ## Run go fmt against code.
+	go fmt ./...
+
+.PHONY: vet
+vet: ## Run go vet against code.
+	go vet ./...
+
+.PHONY:  signimage
+sign-image: ## Build sign-image binary.
```

```suggestion
signimage: ## Build sign-image binary.
```
Are you OK for having the same name in the target and in the PHONY?

---

yes but I'm not sure that rule should be PHONY at all, it does produce a file.... I'll have a think and fix it either way..